packages/react/src/components/RuleBuilder/utils.test.js

```diff
@@ -10,7 +10,7 @@ import {
 let tree;
 
 const NEW_RULE_MATCH = expect.objectContaining({
-  id: expect.stringMatching(/[a-zA-Z0-9]{10}/),
+  id: expect.stringMatching(/[a-zA-Z0-9]{9,10}/),
```

Comment:
There has been another fix just merged that addresses this in a slightly different way, FYI

Reply:
ok fixed this